charts/kubewarden-defaults/Chart.yaml

```diff
@@ -34,7 +34,7 @@ annotations:
   # optional ones:
   catalog.cattle.io/hidden: true # Hide specific charts. Only use on CRD charts.
   catalog.cattle.io/auto-install: kubewarden-crds=1.4.1
-  catalog.cattle.io/upstream-version: 1.7.1
+  catalog.cattle.io/upstream-version: 1.7.3
```

Comment:
heh, this is a fix, seems that in last update we didn't bump the upstream version.